eoscostarica/eden-accounting-standard
# Feat/elected table

### Store Eden Members

### What does this PR do?

- Resolve #8 

### Steps to test

1. `make run`
2. check Eden members with`election_rank` greater than `election_rank` are stored in the `eden_delegates` table

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
